diesel/src/expression/expression_methods/sort_expression_methods.rs

```diff
@@ -0,0 +1,44 @@
+use expression::predicates::*;
+
+#[cfg(feature = "postgres")]
+pub trait Ordering : Sized {
```

Comment:
I don't know.  I thought about it as a trait _on_ ordering, providing methods that work on orderings (it is implemented for Asc and Desc).  But maybe it should be thought of as a trait _for_ where to put the nulls, and in that case `NullsOrdering` (or something similar) is a better name.